scripts/package/package-debian.sh

```diff
@@ -115,10 +115,6 @@ parseargs(){
         echo "Provide the NuGet name of the targetted Shared Framework. Missing option '--framework-nuget-name'" && help
     fi
 
-    if [ -z "$PREVIOUS_VERSION_URL" ]; then
```

Comment:
Upgrade tests are a relic from pre-shared fx. We don't have an upgrade story any more :) just SxS